nservicebus/upgrades/azure-blobstorage-data-bus.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Azure BlobStorage Data Bus upgrade guides
+reviewed: 2022-08-25
+suppressRelated: true
```

Comment:
I can see this pattern working with the evolving systems pages but here this seems like a bad idea. If I add a new upgrade guide I’m not going to know that I need to come here and update this index too. So it won’t get updated - and that’s why the index pages generate their own navigation based on directory traversal anyway.

To add to that, these upgrade guide sections have gotten really out of control anyway. It would probably be better to change these into the proper directory structures for each thing.